### PR TITLE
http jetty example.

### PR DESCRIPTION
**Pull request by rghetia**

### CODE REVIEW

**Comment by rghetia** [comment]
I have compiled and tested with SNAPSHOT version.

**Comment by rghetia** [comment]
missed the copyright statement..

File: examples/src/main/java/io/opencensus/examples/http/jetty/server/HelloWorldServer.java [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+public class HelloWorldServer extends AbstractHandler {
+  private static final Logger logger = Logger.getLogger(HelloWorldServer.class.getName());
+
+  public static class HelloServlet extends HttpServlet {
+    private static String RESOURCE = "<h1>Hello World Servlet Get</h1>";
+
+    private static final long serialVersionUID = 1L;
+
+    private void nonAsyncGet(HttpServletRequest request, HttpServletResponse response)
```
Comment (dinooliva) [inline examples/src/main/java/io/opencensus/examples/http/jetty/server/HelloWorldServer.java]: Consider changing 'nonAsync' to 'blocking'

File: examples/src/main/java/io/opencensus/examples/http/jetty/server/HelloWorldServer.java [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+import org.eclipse.jetty.server.handler.AbstractHandler;
+import org.eclipse.jetty.servlet.ServletContextHandler;
+import org.eclipse.jetty.servlet.ServletHandler;
+
+public class HelloWorldServer extends AbstractHandler {
+  private static final Logger logger = Logger.getLogger(HelloWorldServer.class.getName());
+
+  public static class HelloServlet extends HttpServlet {
+    private static String RESOURCE = "<h1>Hello World Servlet Get</h1>";
+
```
Comment (dinooliva) [inline examples/src/main/java/io/opencensus/examples/http/jetty/server/HelloWorldServer.java]: nit: why is this called RESOURCE?

Comment (rghetia) [reply examples/src/main/java/io/opencensus/examples/http/jetty/server/HelloWorldServer.java]: changed it to body.

File: examples/src/main/java/io/opencensus/examples/http/jetty/server/HelloWorldServer.java (hunk 1 above)
Comment (rghetia) [reply examples/src/main/java/io/opencensus/examples/http/jetty/server/HelloWorldServer.java]: done.

**Pull request merged**